tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/completion/BallerinaCompletionUtils.java

```diff
@@ -95,7 +94,7 @@ private static LookupElementBuilder createLookupElement(@NotNull String name) {
      * @return {@link LookupElementBuilder} which will be used to create the lookup element.
      */
     private static LookupElementBuilder createLookupElement(@NotNull String name,
-                                                            @Nullable InsertHandler<LookupElement> insertHandler) {
+            @Nullable InsertHandler<LookupElement> insertHandler) {
```

Comment:
Earlier formatting looks better. WDYT? Can be updated in few other files as well.

Reply:
+1 :) Reverted formatting.